python/src/trezorlib/cli/__init__.py

```
@@ -212,12 +211,20 @@ def __init__(
     ) -> None:
         self.session_str = session_str
 
-        if session_str is None:
+        decoded_session = None
+        if session_str is not None:
+            try:
+                decoded_session = SessionIdentifier.from_session_str(session_str)
+            except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)